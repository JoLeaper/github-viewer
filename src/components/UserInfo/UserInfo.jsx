import React from 'react';
import PropTypes from 'prop-types';

const UserInfo = props => (
  <>
    <h1>this is where userinfo will go</h1>
  </>
);

UserInfo.propTypes = {
  prop: PropTypes
};

export default UserInfo;
