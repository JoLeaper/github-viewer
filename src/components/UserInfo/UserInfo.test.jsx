import React from 'react';
import { shallow } from 'enzyme';
import UserInfo from './UserInfo';
const userInfoObj =  {
  username: 'JoLeaper',
  followers: 1,
  following: 1,
  githubURL: 'github.com/user/JoLeaper'
};

describe('UserInfo component', () => {
  it('renders UserInfo', () => {
    const wrapper = shallow(<UserInfo userInfo={userInfoObj}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
  
