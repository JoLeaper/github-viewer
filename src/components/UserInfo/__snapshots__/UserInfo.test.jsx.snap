// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo component renders UserInfo 1`] = `
<Fragment>
  <h1>
    Username: 
    JoLeaper
  </h1>
  <h1>
    Followers: 
    1
  </h1>
  <h1>
    Following: 
    1
  </h1>
  <h1>
    githubURL: 
    github.com/user/JoLeaper
  </h1>
</Fragment>
`;
